projects/mysql-server/targets/fuzz_docommand.dict

```diff
@@ -0,0 +1,282 @@
+#
+# AFL dictionary for SQL
+# ----------------------
+#
+# Modeled based on SQLite documentation, contains some number of SQLite
```

Comment:
Probably better not to use a dict with SQLite extensions

Reply:
Done with https://dev.mysql.com/doc/refman/8.0/en/keywords-table.html and https://dev.mysql.com/doc/refman/8.0/en/keywords.html